crates/ruff/src/rules/pylint/rules/too_many_branches.rs

```diff
@@ -205,8 +205,7 @@ else:
     else:
         pass
 "#;
-
-        test_helper(source, 3)?;
+        test_helper(source, 4)?;
```

Comment:
Four is incorrect in my view, there are only 3 branches:

```
if 
else...if 
else...else
```

Reply:
With `elif` it would be, but with the `else` 4 makes more sense (that's different for python than for curly braces languages)